rolling/distribution.yaml

```diff
@@ -1509,6 +1509,12 @@ repositories:
       url: https://github.com/ros2/message_filters.git
       version: master
     status: maintained
+  micro_ros_msgs:
+    release:
```

Comment:
could you add source and doc entries?
it's not a requirement, but it's usually done

Reply:
Done